app/employee-app/backend/modules/database/types.bal

```diff
@@ -261,20 +259,69 @@ type DatabaseConfig record {|
     ConnectionPool connectionPool;
 |};
 
-# [Entity] Sub Team.
-public type SubTeam record {|
-    # Id of the sub team
-    int id;
-    # Title of the sub team
-    string subTeam;
+# Flat list record.
+public type FlatList record {
+    # Business unit array
+    string[] buFlatList;
+    # Team array
+    string[] teamFlatList;
+    # Unit array
+    string[] unitFlatList;
+};
+
+# [LocationData] record.
+public type LocationData record {
+    # Employee location
+    @sql:Column {name: "employee_location"}
+    string location;
+};
+
+# [Entity] Organization data record to set the structure.
+public type OrgData record {|
+    # Name of the business unit
+    string name;
+    # List of teams
+    Team[]? children;
 |};
 
-# [Entity] Team.
-public type Team record {|
-    # Id of the team
-    int id;
-    # Title of the team
-    string team;
-    # List of sub teams
-    SubTeam[]? subTeams;
+# [Database] Organization data.
+public type OrgDataDB record {|
```

Comment:
The naming convention for abbreviations in type/variable names still follows **camelCasing**.
```suggestion
public type OrgDataDb record {|
```